mods/ts/rules/palettes.yaml

```diff
@@ -22,7 +22,6 @@
 		Name: chrome
 		Filename: cameo.pal
 		AllowModifiers: false
-		ShadowIndex: 242
```

Comment:
The proper fix of this would be to decouple the clock palette from the underlaying icon. I had started to work on this, but my last attempt broke support powers. >_>